MdeModulePkg/Core/Dxe/Image/Image.c

```diff
@@ -663,7 +663,16 @@ CoreLoadPeImage (
   // Or, if our memory protection policy specifies that we shouldn't allow such images, return a failure.
   if (!(Image->ImageContext.SupportsNx) && (Image->ImageContext.ImageType == EFI_IMAGE_SUBSYSTEM_EFI_APPLICATION)) {
     if (gDxeMps.ImageProtectionPolicy.Fields.BlockImagesWithoutNxFlag) {
-      return EFI_SECURITY_VIOLATION;
+      DEBUG ((DEBUG_ERROR, "\n"));

```

Comment:
Can you combine this with the next debug print? Newline at the beginning of the string is not much different than the usual convention of at the end in one print.

Reply:
Updated